server/sonar-server/src/main/java/org/sonar/server/es/IndexCreator.java

```diff
@@ -38,32 +42,41 @@
 
   private static final Logger LOGGER = Loggers.get(IndexCreator.class);
 
-  /**
-   * Internal setting stored on index to know its version. It's used to re-create index
-   * when something changed between versions.
-   */
-  private static final String SETTING_HASH = "sonar_hash";
-
+  private final MetadataIndexDefinition metadataIndexDefinition;
+  private final MetadataIndexer metadataIndexer;
+  private final MetadataIndex metadataIndex;
   private final EsClient client;
   private final IndexDefinitions definitions;
 
-  public IndexCreator(EsClient client, IndexDefinitions definitions) {
+  public IndexCreator(EsClient client, IndexDefinitions definitions, MetadataIndexDefinition metadataIndexDefinition, MetadataIndexer metadataIndexer, MetadataIndex metadataIndex) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)